src/components/NavBar/NavBar.styl

```diff
@@ -231,6 +231,11 @@ navbar-width= 17rem;
             box-shadow: 2px 0px 2px 0px rgba(0, 0, 0, 0.16);
         }
 
+        ul#items {
+            margin-left: 0.75rem;
+            width: initial;
```

Comment:
Does this only override the the `width: 100%` on line 205? If so we should just comment that out. Although, I think this has the side effect of limiting the width of the horizontal rule, which I'm not sure if I like better or not. If we want to keep the horizontal rule at full width, we may want to swap the `margin-left:` with a `padding-left` and omit the width limit.